schema/deploy/shipping/views.sql

```diff
@@ -558,3 +560,35 @@ comment on view shipping.observation_with_presence_absence_result_v2 is
   'Joined view of shipping.incidence_model_observation_v3 and shipping.presence_absence_result_v1';
 
 commit;
+
+create or replace view shipping.metadata_for_augur_build_v3 as
+
+    select sample as strain,
+            coalesce(
+              cast(encountered as date),
+              case
+                when date_or_null(sample.details->>'date') <= current_timestamp::date
+                  then date_or_null(sample.details->>'date')
+              end
+            ) as date,
+            'seattle' as region,
+            -- XXX TODO: Change to PUMA and neighborhoods
+            residence_census_tract as location,
+            'Seattle Flu Study' as authors,
+            age_range_coarse,
+            case
+                when age_range_coarse <@ '[0 mon, 18 years)'::intervalrange then 'child'
+                else 'adult'
+            end as age_category,
```

Comment:
I'm also just realizing that this case is wrong. If `age_range_course` is null, then `adult` is returned. The fix is to compare directly to boolean values, e.g.:

```sql
            case age_range_coarse <@ '[0 mon, 18 years)'::intervalrange
                when 't' then 'child'
                when 'f' then 'adult'
            end as age_category,
```

which will only be `child` or `adult` when the age is known (not null) and otherwise be null.

Reply:
Nice catch.